phpdoc.dist.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<phpdoc>
+	<title>WordPress.com VIP – VIP Go Function Documentation</title>
+	<parser>
+		<!-- This places the cache dirs in the root dir `/phpdoc/` for your site -->
+		<target>../../phpdoc/</target>
+		<encoding>utf8</encoding>
+		<markers>
+			<item>TODO</item>
+			<item>FIXME</item>
+		</markers>
+		<visibility></visibility>
+	</parser>
+	<transformer>
+		<!-- This places the docs in the root dir `/phpdoc/` for your site -->
+		<target>../../phpdoc/</target>
```

Comment:
@simonwheatley what's the use case for this path? 

Running locally, it dropped the files + cache in `/Users/nickdaugherty` on my setup - I would have expected `./phpdoc`, with a `.gitignore` entry.

I think the primary use-case for local doc generation will be testing - i.e., as you're working on the mu plugins code, you regenerate the docs prior to committing, to make sure they look right, are free from typos, etc. Keeping them in the same directory probably avoids confusion.